zeq/detail/port.h

```diff
@@ -19,7 +20,7 @@ namespace detail
 uint16_t getPort( const std::string& name );
 
 /** @return a random port number. */
-uint16_t getRandomPort();
+ZEQ_API uint16_t getRandomPort();
```

Comment:
I know it's already merged, but this looks fishy, doesn't it? A function in zeq/detail shouldn't be needed to be exported.